### DISCUSSION TITLE
Skin delete dialog doesn't reduce hitsounds volume during gameplay

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#22053] [Q&A] [answered]
While editing a skin and deleting old versions of it (during gameplay), I noticed that the deletion popup warning doesn't seem to touch the hitsounds, only the music:

https://user-images.githubusercontent.com/43073074/211072194-bcc328b9-ceaf-4860-aee4-90e4d1c102a2.mp4

**Comment by mk56-spn** [comment]
This is probably because hitobjects are part of effects category in audio settings ,and not music. If it did apply the low pass filter  ( I think this is what the effect is called ) to them as it stands It would probably muddy the effects of the overlay aswell. Not sure how feasible it is to avoid this as things are currently set up . Maybe muting them for now or somehow applying the effect locally to the hitobject container

**Reply by frenzibyte** [reply]
I would look at a way of applying the low-pass filter (muting for now) any sound produced by the currently displayed `OsuScreen`, since the dialog *obstructs* the user from interacting with it. I wouldn't spend too much time on this though, it's pretty hidden.

**Reply by peppy** [reply]
This will at very least require nested mixers. It's not going to happen any time soon.

**Reply by soopax** [reply]
The hitsounds also play over track previews in the beatmap listing:

1. Start watching a replay.
2. Open the in-game beatmap listing and click on the play button of any track there.

**Reply by bdach** [reply]
@soopax please open separately as it's most likely going to be a separate fix.

**Reply by soopax** [reply]
@bdach #24170

**Comment by bdach** [comment, marked as answer]
Tracked as issue in https://github.com/ppy/osu/issues/23043
